install/kubernetes/cilium/charts/operator/values.yaml

```diff
@@ -1,4 +1,8 @@
-image: operator
+image:
+  repository: operator
+  tag: latest
+  pullPolicy: Always
+
```

Comment:
All of the other changes were already applied in other PRs, except for this change.

Reply:
(Well, and obviously when this changes the actual daemonset needs to take this into account)

Reply:
We will need to look at this closely from an upgrade instructions angle because changing the "image" value from a string to a map may cause helm to reject the upgrade.